packages/project-access/test/project/find-apps.test.ts

```diff
@@ -289,3 +289,18 @@ describe('Test findFioriArtifacts()', () => {
         ]);
     });
 });
+
+describe('Test findCapProjects()', () => {
+    test('Find CAP projects', async () => {
```

Comment:
@Klaus-Keller Could you please add some nested projects to the test set also, for example see the `cloud-cap-samples` repo : https://github.com/SAP-samples/cloud-cap-samples

Reply:
Added test with and without `srv` folder